server_manager/package.json

```diff
@@ -110,5 +110,8 @@
     "supports-color": "3.1.2",
     "type-detect": "1.0.0"
   },
-  "license": "Apache"
+  "license": "Apache",
```

Comment:
I think this is not the right designation. It may be Apache-2? Also, do we need it on every package.json?